core/src/main/java/org/apache/calcite/sql2rel/StandardConvertletTable.java

```diff
@@ -197,6 +197,8 @@ private StandardConvertletTable() {
         new TimestampDiffConvertlet());
     registerOp(SqlLibraryOperators.TIMESTAMP_SUB,
         new TimestampSubConvertlet());
+    registerOp(SqlLibraryOperators.DATETIME_SUB,
+        new TimestampSubConvertlet());
```

Comment:
move a few lines up, so that it is alphabetical. alphabetical reduces merge conflicts.